### DISCUSSION TITLE
Dispatch multiple events?

### DISCUSSION THREAD

**Discussion by dudelisius** [livewire/livewire#7600] [1. Help]
This question is based on the code that goes with the new screencasts for Livewire 3. Because I do not want to spoil code I will try to describe the question without code examples for now. 

Repository can be found here; https://github.com/livewire/screencast--modals

So I have been following the new screencasts and have a table with users, almost identical to the Post example but with a User model. In the example there is a dropdown for each row where the delete button triggers the delete dialog and the submit button dispatches the deleted event to the parent component. This all works great. 

I want to add a new button to the menu which toggles a status. To do this I figured that dispatching a new event would be the solution. But this breaks the dispatching for both events. 

So my questions are; 

1. Is it possible to dispatch multiple events from within a child component? And so, how? 
2. Is this the best solution to the problem or should I take a different approach? And so, any ideas?

I tried calling the function in the parent component with a `wire:click="$parent.functionIWantToCall()"` and this does work but then the results aren't refreshed in the list like they do on the delete.

If I missed a similar discussion please point me in the right direction.

**Comment by focaicedoc** [comment]
Hard to understand without code. However...

You always can dispatch events to a specific livewire component. 

From Class Component:
`$this->dispatch('event-to-dispatch')->to('LivewireComponent1::class');`
`$this->dispatch('another-event-to-dispatch')->to('LivewireComponent2::class');`
